### PR TITLE
Plural name fixes

### PR DESCRIPTION
**Pull request by BrettDong**
#### Summary
```SUMMARY: None```

#### Purpose of change
Translators pointed out several wrong plural names, such as `faux fur pantss`, `pair of C.R.I.T rec glovess`, etc. because we did not explicitly define their plural names and our script will automatically append a suffix `s` in this case.

#### Describe the solution
Explicitly define their correct plural names.

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -9869,7 +9870,7 @@
     "id": "case_violin",
     "type": "ARMOR",
     "name": "violin case",
-    "name_plural": "violin case",
+    "name_plural": "violin cases",
```
Comment (Night-Pryanik) [inline data/json/items/armor.json]: This is redundant.

**Comment by I-am-Erk** [comment]
Can you please change these to use the new format for names, eg 
```
    "name": { "str": "soap dish", "str_pl": "soap dishes" },
```
I know it's a pain but we're trying to move this way

**Comment by BrettDong** [comment]
> Can you please change these to use the new format for names, eg

I think this is unrelated to the problem I am trying to solve in this PR. And I don't want to make these  namings inconsistent within a single file. It would be better if the transition is made in PRs and whole file by whole file manner.

**Comment by ZhilkinSerg** [comment]
Yeah, we can totally migrate to new format separately.

**Pull request merged**